### PR TITLE
Add missing / at the end of .gitignore line

### PR DESCRIPTION
**Pull request by carakas**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,5 @@
 /bundles/
 /js/
 /css/
-/bin
+/bin/
+
```
Comment (WouterSioen) [inline .gitignore]: why the extra line?

File: .gitignore [hunk 2]
```diff
@@ -17,4 +17,5 @@
 /bundles/
 /js/
 /css/
-/bin
+/bin/
```
Comment (WouterSioen) [inline .gitignore]: for folders, ending with `/*` is even better.

File: .gitignore (hunk 1 above)
Comment (WouterSioen) [inline .gitignore]: while your at it, `/app/sessions` can be removed

File: .gitignore (hunk 2 above)
Comment (carakas) [inline .gitignore]: fixed in ee5a557deb5f231abc61755abae82145d8760050

File: .gitignore (hunk 1 above)
Comment (carakas) [inline .gitignore]: fixed in 46eff83

File: .gitignore [hunk 3]
```diff
@@ -4,17 +4,16 @@
...
-/bin/
-/bundles/
-/js/
-/css/
-/bin
+/bin/*
+/bundles/*
+/js/*
+/css/*
+/bin/*
```
Comment (jeroendesloovere) [inline .gitignore]: This folder is in this file twice...

Comment (WouterSioen) [inline .gitignore]: hmm, good catch.

Comment (carakas) [inline .gitignore]: nice one :)

Comment (WouterSioen) [inline .gitignore]: fixed in 836156f26a0469882cd9d0fd2a2336e279c79120

**Pull request merged**